test/mongoose_graphql_SUITE.erl

```diff
@@ -396,6 +435,36 @@ check_union_permissions(Config) ->
     ?assertPermissionsFailed(Config, FDoc),
     ?assertPermissionsFailed(Config, FDoc2).
 
+%% Domain permissions
+
+check_field_domain_permissions(Config) ->
+    Domain = <<"my-domain.com">>,
+    Config2 = [{op, <<"Q1">>}, {args, #{<<"domain">> => Domain}} | Config],
+    Doc = <<"{ field protectedField }">>,
+    Doc2 = <<"query Q1($domain: String) { protectedField domainProtectedField(argA: $domain"
+             ", argB: \"domain\") }">>,
+    FDoc = <<"{protectedField domainProtectedField(argA: \"domain.com\","
+             " argB: \"domain.com\") }">>,
+    ?assertPermissionsSuccess(Config, Domain, Doc),
+    ?assertPermissionsSuccess(Config2, Domain, Doc2),
+    ?assertDomainPermissionsFailed(Config, Domain, [<<"argA">>], FDoc).
+
+check_child_object_field_domain_permissions(Config) ->
+    Domain = <<"my-domain.com">>,
+    Config2 = [{op, <<"Q1">>}, {args, #{<<"domain">> => Domain}} | Config],
+    Doc = <<"{ obj { field protectedField } }">>,
+    Doc2 = <<"query Q1($domain: String) { obj { protectedField domainProtectedField(argA: $domain"
+             ", argB: \"domain\") } }">>,
+    FDoc = <<"{ obj {protectedField domainProtectedField(argA: \"domain.com\","
+             " argB: \"domain.com\") } }">>,
+    ?assertPermissionsSuccess(Config, Domain, Doc),
+    ?assertPermissionsSuccess(Config2, Domain, Doc2),
+    ?assertDomainPermissionsFailed(Config, Domain, [<<"argA">>], FDoc).
+
+check_interface_field_domain_permissions(_Config) ->
+    %% FIXME provide implementation
```

Comment:
Either remove or comment this test out in the list of test cases, because it is misleading - it always succeeds, but in the report it looks like a successful test case.